packages/target-tauri/runtime-tauri/runtime.ts

```diff
@@ -331,6 +331,27 @@ class TauriRuntime implements Runtime {
     }
     return ''
   }
+  transformStickerURL(sticker_path: string): string {
+    const matches = sticker_path.match(/.*(:?\\|\/)(.+?)\1stickers\1(.*)/)
+    // this.log.info({ transformStickerURL: sticker_path, matches })
+
+    if (matches) {
+      let filename = matches[3]
+      if (decodeURIComponent(filename) === filename) {
+        // if it is not already encoded then encode it.
+        filename = encodeURIComponent(filename)
+      }
+      return `${this.runtime_info?.tauriSpecific?.scheme.stickers}${matches[2]}/${matches[3]}`
+    }
```

Comment:
Can `this.runtime_info?...` ever be `undefined`?

Reply:
TS forced me to add it. in theory it can be undefined, but in that state the app is more broken and we have different problems.